docker/secondary-analysis-python/tools/submit/tests/test_create_analysis_json.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+import os
+import sys
+import json
+
+#pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
Were these comments meant to be removed?

Reply:
Yeah, was an alternative implementation, good catch.